stable/management-ingress/values.yaml

```diff
@@ -29,14 +29,6 @@ tolerations:
   key: node-role.kubernetes.io/infra 
   operator: Exists
 
-cert:
```

Comment:
Have you checked if it is using by `multicloudhub-operator` or not?

Reply:
@clyang82 these values are used as a reference in certificate CR, and will still be created, do you mean we also need to remove below resources?
```
#  oc get certificates.certmanager.k8s.io multicloud-ca-cert
NAME                 READY   SECRET               AGE   EXPIRATION
multicloud-ca-cert   True    multicloud-ca-cert   21h   2026-03-23T08:00:58Z
# oc get issuers.certmanager.k8s.io multicloud-ca-issuer
NAME                   AGE
multicloud-ca-issuer   21h
```

Reply:
That's fine. it should be removed along with cert-manager.